packages/components/src/components/hds/copy/button/index.ts

```diff
@@ -70,23 +88,25 @@ export default class HdsCopyButtonComponent extends Component {
   }
 
   @action
-  onSuccess(args) {
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  onSuccess(args: any) {
```

Comment:
Should I be defining the args here and below using a specific type instead of `any`?

Reply:
we should probably keep it generic as it can have any arguments I assume